videoframes.py

```diff
@@ -123,10 +123,15 @@ class LiveVideoFrame(_VideoFrame):
     Args:
         vlc_instance: VLC instance object.
     """
+
+    stream_end = QtCore.pyqtSignal()
+
     def __init__(self, vlc_instance, stream_url, stream_options, quality):
         super(LiveVideoFrame, self).__init__(vlc_instance)
         self.vlc_instance = vlc_instance
         self.stream = LiveStreamContainer(vlc_instance, stream_url, stream_options, quality)
+        self.stream.on_stream_end = self.stream_end.emit
```

Comment:
The reason I use a signal here is that once again one cannot mutate Qt things from a seperate thread.